starter_code/app.js

```diff
@@ -1,21 +1,48 @@
 
 const express = require('express');
-const hbs     = require('hbs');
+const ejs     = require('ejs');
 const app     = express();
```

Comment:
Try to put all the requires at the top separated from other variables that you add. You will get a cleaner code.